packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -201,6 +202,14 @@ function Navbar() {
     [setLightTheme, setDarkTheme],
   );
 
+  const windowSize = useWindowSize();
+
+  useEffect(() => {
+    if (windowSize === desktopSize && sidebarShown) {
```

Comment:
in the same way `setWindowSize(getSize());` does not trigger useless re-renders, it's not necessary to test `&& sidebarShown` because React will already noop if `setSidebarShown(false);` is called while `sidebarShown=false`.

(also you didn't capture `sidebarShown` in the deps array)

Reply:
https://twitter.com/sebastienlorber/status/1272516868994224128

Reply:
Hum, that's subtle. I updated the code